app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -40,6 +43,13 @@ export const Body = () => {
   }
 
   if (error) {
+    if (!hasRefetched) {
+      setTimeout(() => {
+        refetch();
+        setHasRefetched(true);
+      }, REFETCH_DELAY);
+      return <RefetchAlert />;
+    }
```

Comment:
A few things probably worth calling out:

* This does not catch and retry for JSON.parse-errors only and instead retries if anything fails, but I could change the conditional on L46 to something like `!hasRefetched && error.message.includes("JSON.parse")` if preferred, but not sure if there's ever any other kind error that occurs mid-deploy
* Errors should be captured by Sentry in `ApolloErrorAlert` (if the refetch fails), but not if something fails just once. It didn't seem like we wanted to capture it if it went away after refetch since it's likely a false positive.
* We're also not sending the polling-specific errors to Sentry since it feels edge casey and shouldn't error after repoll, but can if we want to